.gemspec

```diff
@@ -38,5 +38,39 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency(%q<hiera>, [">= 2.0", "< 4"])
   # PUP-7115 - return to a gem dependency in Puppet 5
   # s.add_runtime_dependency(%q<semantic_puppet>, ['>= 0.1.3', '< 2'])
+  # i18n support (gettext-setup and dependencies)
   s.add_runtime_dependency(%q<gettext-setup>, [">= 0.10", "< 1"])
+  s.add_runtime_dependency(%q<locale>, ["~> 2.1"])
+  s.add_runtime_dependency(%q<rake>, ["10.1.1"])
```

Comment:
I've addressed the `rake` dependency situation separately in PR #5784 as it required AppVeyor / Travis changes - we can get that merged in and then I'll rebase and we can get this PR in.  Otherwise, this PR fails on AppVeyor when `rake` is moved to a `development` dependency.